Detectors/TRD/simulation/src/TrapSimulator.cxx

```diff
@@ -316,7 +320,7 @@ void TrapSimulator::printFitRegXml(ostream& os) const
 void TrapSimulator::printTrackletsXml(ostream& os) const
```

Comment:
Wow, I wasn't aware this is in the TRAPsimulator. This looks like it replicates the output that one could get from a chamber running with a FeeServer. Could be useful, if we invest the time to set it up, e.g. at UCT.
However, it is somewhat complementatry to the tests that Venelin can do with a single readout board, and it is also not needed if we have optical readout of the chamber, e.g. at CERN.

For now, this is just a comment. If it is too much work to keep it up to date, it could be dropped.

Reply:
It should be correct. The only possible error is the assumption that the 0th cpu contains the header. I suspect this might not be reflective of the MCM. That assumption is made throughout though. It should not affect the output though.